### PR TITLE
Change LICENSE to BUSL-1.1

### PR DESCRIPTION
**Pull request by wkozyra95**

### CODE REVIEW

File: docs/src/pages/index.tsx [hunk 1]
```diff
@@ -452,11 +452,11 @@ function ContactUs() {
...
               <p className={styles.sectionSubheading}>
-                LiveCompositor is free and open-source. It's created by Software Mansion - a
-                software company that is specialized in building tools for developers. At Software
-                Mansion, we work on multiple multimedia projects, like Membrane Framework, Elixir
-                WebRTC, FishJam, and more. We also work on custom solutions for clients. Contact us
-                and create something together.
+                LiveCompositor is developed by Software Mansion - a software company that is
+                specialized in building tools for developers. At Software Mansion, we work on
+                multiple multimedia projects, like Membrane Framework, Elixir WebRTC, FishJam, and
+                more. We also work on custom solutions for clients. Contact us and create something
```
Comment (darthez) [inline docs/src/pages/index.tsx:458]: Suggestion: maybe it will be worthwhile to add here explicit email address - projects@swmansion.com

**Review by WojciechBarczynski — APPROVED** [review]
We need to change content on the website and modify some statements like "free and open-source" after this change. idk if you want to do it in this or a separate PR, but it has to be done

**Comment by wkozyra95** [comment]
> We need to change content on the website and modify some statements like "free and open-source" after this change. idk if you want to do it in this or a separate PR, but it has to be done


Where? did I miss any?

File: docs/src/pages/index.tsx (hunk 1 above)
Comment (wkozyra95) [reply docs/src/pages/index.tsx:458]: I'll add this in follow up.

**Pull request merged**